tests/autoyast/autoyast_verify.pm

```diff
@@ -92,7 +89,7 @@ sub test_flags {
     # 'fatal' - whole test suite is in danger if this fails
     # 'milestone' - after this test succeeds, update 'lastgood'
     # 'important' - if this fails, set the overall state to 'fail'
-    return {fatal => 1};
+    return {important => 1};
```

Comment:
It's true that I did not think about rollback, but now I think that it is not a problem in this particular case. 

The autoyast_verify just verifies that everything is configure correctly, it does not produce log entries by itself. If it fails then I am interested in the logs from installation, which are preserved.